pkg/cluster/cluster.go

```diff
@@ -399,7 +399,7 @@ func CreateCluster(cmv1Client *cmv1.Client, config Spec, dryRun bool) (*cmv1.Clu
 }
 
 func isGCPNetworkExists(existingVPC ExistingVPC) bool {
-	return existingVPC.VPCName != "" && existingVPC.ControlPlaneSubnet != "" && existingVPC.ComputeSubnet != ""
+	return existingVPC.VPCName != "" || existingVPC.ControlPlaneSubnet != "" || existingVPC.ComputeSubnet != ""
```

Comment:
Wouldn't this change cause the cluster builder to try building the cluster with the missing attributes and only fail then? 
Perhaps instead of using this method for [checking if we want to add these attributes](https://github.com/zvikorn/ocm-cli/blob/2e8877639fb3e90c90640346432e95bde7cc1bdc/pkg/cluster/cluster.go#L333) to the builder, we need to check if any of the attributes is missing and error early?

Reply:
we build the GCPNetwork object if at least one GCPNetwork attribute exists. If CS needs all attributes it will return an error.